.github/workflows/master_fuzzer_binaries.yml

```diff
@@ -29,8 +29,9 @@ jobs:
         with:
           crate: cargo-bolero
           # TODO: remove the below once https://github.com/camshaft/bolero/pull/195 is released on crates.io
-          git: https://github.com/camshaft/bolero
-          rev: 8c5a50a57b0e4c4cc8111cfd95670dc75cd2dea7
+          # and https://github.com/camshaft/bolero/pull/196 has a proper fix
```

Comment:
Nit: Looks like 195 is going to be released well before 196 is fixed, so perhaps just refer to that one? But it is fine either way.